server/src/api/graphql/schema.graphql

```diff
@@ -0,0 +1,52 @@
+type Author {
+  id: String
+  name: String
+  biography: String
+  bornDate: String
+  books: [Book]
+  totalBooks: Int
+  createdAt: String
+  updatedAt: String
+}
+
+type Book {
+  id: String
+  title: String
+  description: String
+  publishedDate: String
+  authorId: String
```

Comment:
Why is authorId separately required?

Reply:
It's not required, but it's present because we have this piece of information in the database schema.